app/models/concerns/request_helper.rb

```diff
@@ -27,7 +27,8 @@ def get_html(url, set_error_callback, header_options = {}, force_proxy = false)
         end
         get_html(url, set_error_callback, header_options, true)
       rescue Net::HTTPClientException => e
-        handle_http_exception_error(e)
+        set_error_callback.call(message: 'Proxy Error', code: Lapis::ErrorCodes::const_get('LOGIN_REQUIRED'))
```

Comment:
@caiosba I have one question about this one: I left the Lapis error code as `LOGIN_REQUIRED`, but I'm not sure that is the best code to use.

Reply:
I was thinking about that yesterday. I suggest you add a new `PROXY_ERROR` to `lib/lapis/error_codes.rb`, since it's a core feature of the application.

Reply:
Will do! Another question: is there a logic to follow behind the numbers for each code?

Reply:
No, it's just incremental (some are skipped because they were deleted, I think).